dom/manipulate.ts

```diff
@@ -2,6 +2,20 @@ import {isElement} from "./utils";
 import {setAttrs} from "./attr";
 import {setStyles} from "./css";
 
+/**
+ * Options for createElement()
+ */
+export type CreateElementOptions = mojave.types.KeyMap & {
```

Comment:
We can just inline the KeyMap. I only created the `KeyMap` type, because I didn't understood the object syntax.

You can just do:

```ts
export type CreateElementOptions = {
    [key: string]: string | number,
    css? : {[key: string]: string | number},
    text? : string,
    html? : string,
}
```